Dockerfile

```diff
@@ -1,14 +1,12 @@
-FROM openjdk:8-jdk-alpine
+FROM openjdk:8-jre-alpine
 
 # Set working directory
 WORKDIR /app
 
-# Copy current directory contents into container
-COPY . /app
+# Copy jar file into container
+COPY ./SNAPSHOT.jar /app
 
 # port to expose
 EXPOSE 80
```

Comment:
Do we need to expose this particular port. What port(s) is the application listening on?